README.md

```diff
@@ -45,23 +47,58 @@ The webhook application can be configured using the flags outlined below.
+--no-tls                                                             Do not use TLS.
+--rule-resource-limit-cpu-must-be-nonzero                            Whether 'cpu' limit in resource specifications must be a nonzero value.
+--rule-resource-limit-cpu-required                                   Whether 'cpu' limit in resource specifications is required.
+--rule-resource-limit-memory-must-be-nonzero                         Whether 'memory' limit in resource specifications must be a nonzero value.
+--rule-resource-limit-memory-required                                Whether 'memory' limit in resource specifications is required.
+--rule-resource-request-cpu-must-be-nonzero                          Whether 'cpu' request in resource specifications must be a nonzero value.
+--rule-resource-request-cpu-required                                 Whether 'cpu' request in resource specifications is required.
+--rule-resource-request-memory-must-be-nonzero                       Whether 'memory' request in resource specifications must be a nonzero value.
+--rule-resource-request-memory-required                              Whether 'memory' request in resource specifications is required.
+--rule-security-readonly-rootfs-required                             Whether 'readOnlyRootFilesystem' in security context specifications is required.
+--rule-security-readonly-rootfs-required-whitelist-enabled           Whether rule 'readOnlyRootFilesystem' in security context can be ignored by container whitelisting.
+--rule-resource-violation-message                                    Additional message to be included whenever any of the resource-related rules are violated.
+--rule-ingress-collision                                             Whether ingress tls and host collision should be checked 
+--rule-ingress-violation-message                                     Additional message to be included whenever any of the ingress-related rules are violated.
+--tls-cert-file string                                               Path to the certificate file. Required, unless --no-tls is set.
+--tls-private-key-file string                                        Path to the certificate key file. Required, unless --no-tls is set.
+--annotations-prefix                                                 What prefix should be used for admission validation annotations.
 ```
 
+In case you want to check `readOnlyRootFilesystem` property globally but also allow some containers requiring writable root filesystem, they can be whitelisted by using annotations.
+In order to do that you need to:
+* Allow whitelisting of containers for this option by `--rule-security-readonly-rootfs-required-whitelist-enabled` option
+* Provide annotation `readonly-rootfs-containers-whitelist` at Pod level (by default, this annotation does not have prefix)
+* If you want to add prefix for all admission validation annotations, use `--annotations-prefix` option
+
+Example:
```

Comment:
I'm not particularly fond of this section, it's too complicated. I think it would be much simpler to have a default annotation prefix, use that prefix in all examples, and then document that the prefix can be changed.

Reply:
@Flektoma will change this. We will introduce some default prefix.

Reply:
@petr-k any suggestions? My proposal is `admission.validation.avast.com/....`.

Reply:
I'm ok with this naming.